main.js

```diff
@@ -22,18 +32,57 @@ async function main() {
                 branchesToDelete.push(pull.data.head.ref)
             }
         }
-
+        
+        if (prefix) {
```

Comment:
I don't know why you did it like this. Why `if prefix` here and not later, how it was done before? Why prefix is handled different than suffix?

Reply:
We can make suffix be in parity with the prefix. Honestly, I have not experienced repos that name git branches with suffixes, so I left it be